docs/user-guides/server.md

```diff
@@ -60,19 +60,19 @@ The procedure and UI of ONNX and TensorRT runtime would look the same as Pytorch
 
 ## Model support
 
-Open AI has released 9 models so far. `ViT-B/32` is used as default model in all runtimes. Due to the limitation of some runtime, not every runtime supports all nine models. Please also note that different model give different size of output dimensions. This will affect your downstream applications. For example, switching the model from one to another make your embedding incomparable, which breaks the downstream applications. Here is a list of supported models of each runtime and its corresponding size:
+Open AI has released 9 models so far. `ViT-B/32` is used as default model in all runtimes. Due to the limitation of some runtime, not every runtime supports all nine models. Please also note that different model give different size of output dimensions. This will affect your downstream applications. For example, switching the model from one to another make your embedding incomparable, which breaks the downstream applications. Below is a list of supported models of each runtime and its corresponding size. We also include a sample disk usage and peak memory usage when running with a single Nvidia TITAN RTX GPU with 24 GB memory and with a batch size of 256.
 
-| Model | PyTorch |  ONNX | TensorRT |  Output dimension | 
-| --- |---------| ---- | --- |--- |
-| RN50 | ✅ |✅ | ✅| 1024 | 
-| RN101 | ✅ |✅ | ✅| 512 | 
-| RN50x4 | ✅ |✅ | ✅| 640 |
-| RN50x16 | ✅ |✅ | ❌| 768 |
-| RN50x64 | ✅ |✅ | ❌| 1024 |
-| ViT-B/32  | ✅ |✅ | ✅| 512 |
-| ViT-B/16 | ✅ |✅ | ✅| 512 |
-| ViT-L/14 | ✅ |✅ | ✅| 768 |
-| ViT-L/14-336px | ✅ |✅ | ❌| 768 |
+| Model          | PyTorch | ONNX | TensorRT | Output Dimension | Disk Usage (MB) | Peak RAM Usage (GB) | Peak VRAM Usage(GB) |
+|----------------|---------|------|----------|------------------|-----------------|---------------------|---------------------|
+| RN50           | ✅       | ✅    | ✅        | 1024             | 256             | 2.97                | 4.25                |
+| RN101          | ✅       | ✅    | ✅        | 512              | 292             | 3.05                | 4.28                |
+| RN50x4         | ✅       | ✅    | ✅        | 640              | 422             | 3.18                | 6.92                |
+| RN50x16        | ✅       | ✅    | ❌        | 768              | 661             | 3.55                | 13.01               |
+| RN50x64        | ✅       | ✅    | ❌        | 1024             | 1382            | 3.95                | 20.26               |
+| ViT-B/32       | ✅       | ✅    | ✅        | 512              | 351             | 3.18                | 2.19                |
+| ViT-B/16       | ✅       | ✅    | ✅        | 512              | 354             | 3.17                | 3.90                |
+| ViT-L/14       | ✅       | ✅    | ✅        | 768              | 933             | 3.64                | 5.38                |
+| ViT-L/14-336px | ✅       | ✅    | ❌        | 768              | 934             | 3.67                | 11.36               |
```

Comment:
on RAM and vRAM, either the number is wrong or the column names are wrong. Please check again

Reply:
I quickly reran the test and noticed that the RAM usage is under-calculated by ~20%. The table is updated